src/tpm/kmyth_seal_unseal_impl.c

```diff
@@ -523,36 +621,35 @@ int tpm2_kmyth_unseal(uint8_t * input,
 //############################################################################
 // tpm2_kmyth_seal_file()
 //############################################################################
-int tpm2_kmyth_seal_file(char *input_path,
+int tpm2_kmyth_seal_file(char * input_path,
                          uint8_t ** output,
-                         size_t *output_len,
-                         uint8_t * auth_bytes,
-                         size_t auth_bytes_len,
-                         uint8_t * owner_auth_bytes,
-                         size_t oa_bytes_len,
-                         int *pcrs, size_t pcrs_len, char *cipher_string,
-                         char *expected_policy, bool bool_trial_only)
+                         size_t * output_len,
+                         char * auth_string,
+                         char * owner_auth_string,
+                         char * cipher_string,
+                         PCR_SELECTIONS * pcrs_in,
+                         TPML_DIGEST * digests_in,
+                         bool bool_trial_only)
 {
-  uint8_t* data = NULL;
+  uint8_t * data = NULL;
   size_t data_len = 0;
   
-  // Only validate the input if we're not just checking the current
-  // PCR values.
+  // validate the input, unless just computing policy digest value
   if(!bool_trial_only)
   {
     // Verify input path exists with read permissions
     if (verifyInputFilePath(input_path))
     {
-      kmyth_log(LOG_ERR, "input path (%s) is not valid ... exiting", input_path);
+      kmyth_log(LOG_ERR, "input path (%s) is not valid", input_path);
       return 1;
     }
 
     if (read_bytes_from_file(input_path, &data, &data_len))
     {
-      kmyth_log(LOG_ERR, "seal input data file read error ... exiting");
+      kmyth_log(LOG_ERR, "seal input data file read error");
       if (data != NULL)
       {
-	free(data);
+	      free(data);
```

Comment:
Extra whitespace.

Reply:
I think I fixed this. I looked fine in my editor (VS Code), both before and after commit [8bc5da2](https://github.com/NationalSecurityAgency/kmyth/pull/187/commits/8bc5da26598a0cc0959dc0222aa71e4d48d29c4d), but 'git diff' indicated the removal of the extra whitespace.